include/avif/avif.h

```diff
@@ -1174,6 +1174,23 @@ typedef enum avifProgressiveState
 } avifProgressiveState;
 AVIF_API const char * avifProgressiveStateToString(avifProgressiveState progressiveState);
 
+// Types of image content that can be encoded/decoded.
+typedef enum avifImageContentTypeFlag
+{
+    AVIF_CONTENT_NONE = 0,
+    AVIF_CONTENT_COLOR = (1 << 1),
+    AVIF_CONTENT_ALPHA = (1 << 2),
+#if defined(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
+    AVIF_CONTENT_GAIN_MAP = (1 << 3),
+    AVIF_CONTENT_ALL = AVIF_CONTENT_COLOR | AVIF_CONTENT_ALPHA | AVIF_CONTENT_GAIN_MAP,
+#else
+    AVIF_CONTENT_ALL = AVIF_CONTENT_COLOR | AVIF_CONTENT_ALPHA,
+#endif
+
+    AVIF_CONTENT_DECODE_DEFAULT = AVIF_CONTENT_COLOR | AVIF_CONTENT_ALPHA,
```

Comment:
I thought it was better to have DECODE in the name in case this enum is used for other things in the future. This is also why I didn't use "decode" in the name of the enum.